test/ufuzz.js

```diff
@@ -135,23 +182,25 @@ var FUNC_TOSTRING = [
     "}();",
     ""
 ].join("\n");
+var loops = 0;
+var funcs = 0;
 
 function run_code(code) {
     var stdout = "";
     var original_write = process.stdout.write;
-    process.stdout.write = function(chunk) {
+    process.stdout.write = function (chunk) {
```

Comment:
Sorry about the confusion. 

What I meant in this particular case is don't change any whitespaces except for the indentation.

There are a few other places where this happened as well.